pro/src/pages/CollectiveOfferFromRequest/CollectiveOfferFromRequest.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+
+import { GetCollectiveOfferRequestResponseModel } from 'apiClient/v1'
+import ActionsBarSticky from 'components/ActionsBarSticky'
+import { SummaryLayout } from 'components/SummaryLayout'
+import {
+  CollectiveOfferTemplate,
+  createOfferFromTemplate,
+} from 'core/OfferEducational'
+import getCollectiveOfferTemplateAdapter from 'core/OfferEducational/adapters/getCollectiveOfferTemplateAdapter'
+import useNotification from 'hooks/useNotification'
+import { Button, Title } from 'ui-kit'
+import Spinner from 'ui-kit/Spinner/Spinner'
+import { getDateToFrenchText } from 'utils/date'
+
+import getOfferRequestInformationsAdapter from './adapters/getOfferRequestInformationsAdapter'
+import styles from './CollectiveOfferFromRequest.module.scss'
+
+const CollectiveOfferFromRequest = (): JSX.Element => {
+  const navigate = useNavigate()
+  const notify = useNotification()
+
+  const [informations, setInformations] =
+    useState<GetCollectiveOfferRequestResponseModel | null>(null)
+  const [offerTemplate, setOfferTemplate] = useState<CollectiveOfferTemplate>()
+  const [isLoading, setIsLoading] = useState(true)
+
+  const { offerId, requestId } = useParams<{
+    offerId: string
+    requestId: string
+  }>()
+
+  const handleButtonClick = () => {
+    createOfferFromTemplate(navigate, notify, Number(offerId))
+  }
+
+  const fetchOfferTemplateDetails = async () => {
+    if (offerId) {
+      const { isOk, payload, message } =
+        await getCollectiveOfferTemplateAdapter(Number(offerId))
+      if (!isOk) {
+        return notify.error(message)
+      }
+      setOfferTemplate(payload)
+    } else {
+      return notify.error(
+        'Une erreur est survenue lors de la récupération de votre offre'
+      )
+    }
+  }
+
+  const getOfferRequestInformation = async () => {
+    if (requestId) {
+      const { isOk, message, payload } =
+        await getOfferRequestInformationsAdapter(Number(requestId))
+
+      if (!isOk) {
+        return notify.error(message)
+      }
+      setInformations(payload)
+      setIsLoading(false)
+    }
+  }
+
+  useEffect(() => {
+    fetchOfferTemplateDetails()
+    getOfferRequestInformation()
```

Comment:
Proposition. Et comme ça tu peux enlever les `if` des deux méthodes.

```suggestion
    offerId && fetchOfferTemplateDetails()
    requestId && getOfferRequestInformation()
```